src/Spice86.Shared/Emulator/VM/Breakpoint/Serializable/SerializedBreakpoints.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Spice86.Shared.Emulator.VM.Breakpoint.Serializable;
+/// <summary>
+/// Container class for a collection of serialized breakpoints.
+/// </summary>
+public record SerializedBreakpoints {
+    /// <summary>
+    /// Gets the format version.
+    /// </summary>
+    public int Version { get; init; } = 1;
```

Comment:
Same here. I would put the version on the container.

Reply:
I removed it.